index.md

```diff
@@ -37,6 +37,34 @@ consumers of your library:
 A scope may be provided to a commit's type, to provide additional contextual information and
 is contained within parenthesis, e.g., `feat(parser): add ability to parse arrays`.
 
+## Examples
+
+### Commit message with description and breaking change in body
+```
+feat: allow provided config object to extend other configs
+
+BREAKING CHANGE: `extends` key in config file is now used for extending other config files
+```
+
+### Commit message with no body
+```
+docs: correct spelling of CHANGLOG
+```
+
+### Commit message with scope
+```
+feat(lang): added polish language
+```
+
+### Commit message for a fix using an (optional) issue number.
+```
+fix: minor typos in code
+
+see the issue for details on the typos fixed
+
+fixes issue #12
```

Comment:
This is typically `Closes: #12`. The body above should start with caps.

Reply:
@blackxored those aren't on the specs.

Reply:
They might not be, but I think we should strive for good commit messages as examples, and this isn't one IMHO, at least for the examples providing one that wouldn't clash with good practices (for ex. commitlint and config-conventional which people will learn to associate with this even if they're not).
In that spirit, I'd replace the body and make the footer just `Closes: #12`.

Reply:
I may agree with you, but if isn't specified in the specs it won't be used.
You can open another PR and propose those changes and discuss it, but atm those are just your usage of the specs.

Reply:
Hmm, as I've pointed out in https://github.com/conventional-changelog/conventionalcommits.org/pull/21#discussion_r176935181 I think it's a natural step for people to refer to @commitlint/config-conventional while adopting this spec, it's not just my usage but what's expected by the ecosystem I reckon, commitlint would error at these commit messages, and it's counterintuitive IMHO. Thoughts?

Reply:
The point is:

If we want to add this to the spec we should open issues -> discuss it -> create PR improving the specs.

Reply:
Since my examples follow the spec, I think they should stay as-is and be merged and another issue should be opened to discuss whether or not the examples should demonstrate more than the spec and why. I typically don't like the examples to include anything other than exactly what the spec requires. If the spec is missing some best practices, the spec itself should be improved, and that's a separate discussion.

Reply:
Yup, avoiding to block a PR and to delay the merge for discussions.